spi/src/main/java/org/jboss/pnc/spi/executor/BuildExecutionConfiguration.java

```diff
@@ -58,9 +58,11 @@
 
     Map<String, String> getGenericParameters();
 
-    boolean isTempBuild();
+    @Override
+	boolean isTempBuild();
 
-    String getTempBuildTimestamp();
+    @Override
+	String getTempBuildTimestamp();
```

Comment:
ehm, broken formatting, but thanks for adding the Override. I forgot obviously.

Reply:
Override is added automatically by eclipse, you should migrate back ;-) But the automatic addition used again a tab instead of 4 spaces :-/